### DISCUSSION TITLE
Displaying Math Equation

### DISCUSSION THREAD

**Discussion by Basavaraj-PN** [openframeworks/openFrameworks#7297] [Q&A]
Is there way to display math equation properly? may be "ofxLatex.h" or MathJax support?

**Comment by kylerneva** [comment]
At the moment, openFrameworks doesn’t have built-in support for rendering math equations like LaTeX or MathJax.
	•	MathJax is web-based, so it won’t work directly unless you embed a web view.
	•	There’s no official ofxLatex addon, but some people generate equations as SVG/PNG (via LaTeX) and then load them as textures in OF.
	•	Another option is to use a font that includes common math symbols and render simple formulas manually.

So yes, it’s possible, but usually handled outside OF (pre-rendering) rather than inside the framework itself.

Hope this helps 🙂
